code/src/constants/common.ts

```diff
@@ -17,3 +17,8 @@ export const ROLES = {
   SHOP_MANAGER: "ROLE_MANAGER",
   ADMIN: "ROLE_ADMIN"
 } as const;
+
+export const USER_STATUSES = {
+  ACTIVE: "ACTIVE",
+  DEACTIVATED: "DEACTIVATED"
+} as const;
```

Comment:
In general what do you think about `enum` approach ?
I thought about it but didn't catch why `const` obj with properties better ?
What do you think ?

Reply:
Discussed